.gitignore

```diff
@@ -41,3 +41,4 @@ Brewfile.lock.json
 .github/actions/simplecov-report/lib/
 test/minitest/minitest_time_report
 gem_manifest_*.json
+nr-security-home
```

Comment:
Do we still need this? I thought [NEWRELIC-8432](https://issues.newrelic.com/browse/NEWRELIC-8432) moved the directory into "log"?

Reply:
The cross-agent issue has been raised. As of this time, we're in compliance with how things work for everyone, so no changes to be made just now. We'll handle Ruby specific changes should the spec discussion lead to a change.